.eslintrc

```diff
@@ -21,14 +27,17 @@
   "plugins": [
     "react", "import"
   ],
+  "parser": "babel-eslint",
   "settings": {
     "import/parser": "babel-eslint",
     "import/resolve": {
       moduleDirectory: ["node_modules", "src"]
-    }
+    },
+    "import/ignore": [".scss$", ".css$", ".svg$"]
   },
   "globals": {
     "__DEVELOPMENT__": true,
+    "__PRODUCTION__": true,
     "__CLIENT__": true,
     "__SERVER__": true,
```

Comment:
I think `__DEVELOPMENT__`, `__SERVER__`, and `__DISABLE_SSR__` are no longer valid. These are defined in the webpack configuration... Could you please remove them?